### ISSUE TITLE
External webcam extremely dark despite changing exposure values

### ISSUE THREAD

**Issue by SussyImp0ster** [freemocap/freemocap#616]
I just bought a Nexigo N60 to use for freemocap in conjunction with my built in laptop camera, but for some reason its brightness won't change even if I adjust the exposure; however, the brightness on the built-in webcam works just fine. I tried values up to 100 with no effect on the Nexigo webcam.

**Comment by philipqueen** [comment]
Hi @SussyImp0ster, just to be sure, did you click the "Apply Settings to Cameras" button after adjusting the exposure?

**Comment by SussyImp0ster** [comment]
@philipqueen Yes, I made sure to do that. I looked through past issues and found someone who had a similar issue a while ago, so I think it might be a bug with certain external webcams.

**Comment by philipqueen** [comment]
ok thanks for checking! We're in the process of rebuilding the camera backend, so I'll test if this behaviour is present there.

**Comment by SussyImp0ster** [comment]
Great, thanks! Looking forward to an update.
